mslib/mscolab/file_manager.py

```diff
@@ -250,6 +258,74 @@ def modify_user(self, user, attribute=None, value=None, action=None):
             db.session.commit()
         return True
 
+    def delete_user_profile_image(self, image_to_be_deleted):
```

Comment:
Just a sidenote, nothing to do now: since this method is named `delete_user_profile_image` I think it would semantically make more sense to pass a user to this method, instead of a path.

It works as-is though, so this is something that could be refactored in the future.

Reply:
Okay